ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -1622,6 +1622,22 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-azure-ipi-confidential-trustedlaunch
+- as: azure-ipi-64k-pages-confidential-trustedlaunch-f28
```

Comment:
This should be a multi job as per requests in MCO-800, is there anything else you want to test on amd64 about it?

Reply:
the job in this PR is just for testing purpose, we just provide the step.

Reply:
the mcp step is common, it should work for both amd64 and arm64. and the step can be used in any install provision chain

Reply:
sure, it seems so.
However, if you can add a test config in the nightly file for multi-arch by just copying and pasting a similar job to this here, we're just done with the CI integration task while the testing proceeds.

Reply:
for CI integration, there are 2 phases
1. design and impl mco steps for custom mcp and 64kps kernel type (1st PR)
2. sync with multi-arch team about how many arm64 based workflows need to be customized (2nd PR)